applications/search/src/pages/search-page/filter-pills/filter-pills.component.jsx

```diff
@@ -20,18 +20,27 @@ const getFilterLabel = (key, keyValue, publishers) => {
     return localization.unknown;
   }
   if (key === 'orgPath') {
-    return (
-      getTranslateText(_.get(publishers, [keyValue, 'prefLabel'])) ||
-      _.get(publishers, [keyValue, 'name'])
-    );
+    const currentPublisher = publishers[keyValue];
+    if (currentPublisher) {
+      return currentPublisher.id === 'STAT' ||
```

Comment:
alternativ kunne være at du bare prøver å finne med currentPublisher.name, of hvis ikke finnes, bruker fallback method:
```
localization.facet.publishers[currentPublisher.name] ||
 getTranslateText(_.get(publishers, [keyValue, 'prefLabel'])) ||
            _.capitalize(_.get(publishers, [keyValue, 'name'], keyValue));
```
også det ser ut litt feil, at if har ikke fallback, hvis currentPublisher finnes ikke.

Reply:
rettet